backstop/assets/theme.js

```diff
@@ -1,4 +1,4 @@
-const THEMEAVAILABLE = ['iui-theme-light', 'iui-theme-dark'];
+const THEMEAVAILABLE = ['iui-theme-light', 'iui-theme-dark', 'iui-theme-light-hc', 'iui-theme-dark-hc'];
```

Comment:
Would it make sense to have another button to reach those themes faster...?

Reply:
And all our test images changes 😕 Maybe right-click could expand dropdown with all themes.

Reply:
Most of our test images should take sections with components, not full page, I think...

Reply:
For test images that do have the theme button within, we should be using something like `hideSelectors: ['theme-button'],` in the scenario JS.

Reply:
see #511